tiup/tiup-component-cluster-clean.md

```diff
@@ -6,34 +6,39 @@ title: tiup cluster clean
 
 在测试环境中，有时候需要将集群重置回刚部署的状态，即删除所有数据，命令 `tiup cluster clean` 可以很方便的做到这一点：它会停止集群，然后删除集群上的数据。手工重启集群之后，就能得到一个全新的集群了。
 
-> **注意：**
+> **警告：**
 >
 > 该命令一定会先停止集群（即使选择只清理日志也是），生产环境请勿使用。
 
 ## 语法
 
-```sh
+```shell
 tiup cluster clean <cluster-name> [flags]
 ```
 
 `<cluster-name>` 为要清理的集群。
 
 ## 选项
 
-### --all（boolean，默认 false）
+### --all
 
-同时清理数据和日志，等价于同时指定 `--data` 和 `--log`，若不指定该选项，则必须至少指定以下选项之一：
+- 同时清理数据和日志，等价于同时指定 `--data` 和 `--log`，若不指定该选项，则必须至少指定以下选项之一：
 
-- --data：清理数据
-- --log：清理日志
+    - --data：清理数据
+    - --log：清理日志
 
-### --data（boolean，默认 false）
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
+
+### --data
 
 该选项开启数据清理，若不指定该选项，也不指定 `--all`，则不清理数据。
 
-### --log（boolean，默认 false）
+### --log
 
-该选项开启日志清理，若不指定该选项，也不指定 `--all`，则不清理日志。
+- 该选项开启日志清理，若不指定该选项，也不指定 `--all`，则不清理日志。
+- 数据类型：`BOOLEAN`
+- 该选项默认关闭，默认值为 `false`。在命令中添加该选项，并传入 `true` 值或不传值，均可开启此功能。
 
 ### --ignore-node（StringArray，默认为空）
```

Comment:
Should the info here about the data type and default value be updated to an unordered list too?

Reply:
There are hundreds of similar modifications to make in the TiUP docs and the workload can be huge. This PR only updates/refines a part of them. Other unrefined titles will be refined by upcoming PRs.